addons/mail/models/mail_thread.py

```diff
@@ -266,8 +247,7 @@ def create(self, vals_list):
         if not self._context.get('mail_create_nosubscribe'):
             for thread in threads:
                 self.env['mail.followers']._insert_followers(
-                    thread._name, thread.ids, self.env.user.partner_id.ids,
-                    None, None, None,
+                    thread._name, thread.ids, self.env.user.partner_id.ids, None,
```

Comment:
Maybe use kwargs? All of those param (especially None) are confusing for someone not used to the signature of that method.

Reply:
> Maybe use kwargs? All of those param (especially None) are confusing for someone not used to the signature of that method.

still open comment

Reply:
Good idea. Done.